src/components/Onboard/SerialPortOption.tsx

```diff
@@ -70,8 +69,10 @@ const SerialPortOption = ({
               </h1>
             </div>
             <h1 className="pl-6 pr-2 ml-4 text-sm leading-5 font-light text-red-600 mt-0.5">
-              Could not connect to serial port on {name}, failed with error “
-              {connection_error}”
+              Could not connect to serial port on {name}, failed with error
+              &quot;
```

Comment:
When I tried this, it never actually showed; the loading would just go on indefinitely until the application was F5 refreshed or closed. I'd suggest adding a timeout